src/lib/skilling/skills/crafting/craftables/built.ts

```diff
@@ -11,21 +11,6 @@ const Built: Craftable[] = [
 		inputItems: resolveNameBank({ 'Serpentine visage': 1 }),
 		tickRate: 3
 	},
-	{
-		name: 'Slayer helmet',
-		id: itemID('Slayer helmet'),
-		level: 55,
-		xp: 0,
-		inputItems: resolveNameBank({
-			'Black mask': 1,
-			Earmuffs: 1,
-			Facemask: 1,
-			'Nose peg': 1,
-			'Spiny helmet': 1,
-			'Enchanted gem': 1
-		}),
-		tickRate: 3
```

Comment:
This removes the ability to craft a slayer hem? im confused

Reply:
slayer helm isn't crafted, it's created (instantly)

but i did forget the 55 crafting requirement, i'll add that to the creatable.